csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -151,7 +151,7 @@ function setUpMode(){
 
```

Comment:
Also could you see if you can add functionality to let the user pick the mode they use if it's loaded without url parameters? So like 2d-shape-manipulations users are held to a specific mode if the interactive is opened in the chapter, but if they open it from the 'interactives' page they get a dropdown or something so they can load the other modes

Reply:
I'll make an issue for this if that's ok